plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -1362,6 +1378,31 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             (\() -> [] @(Data,Data))
             (runCostingFunOneArgument . paramMkNilPairData)
+    -- Bitwise
+    toBuiltinMeaning _ver IntegerToByteString =
+        makeBuiltinMeaning integerToByteString mempty
```

Comment:
Is `mempty` here correct @michaelpj ?

Reply:
It's where the costing function needs to go, so yes for now if we're going to cost it, which I guess we are.